lint.sh

```diff
@@ -7,6 +7,9 @@ pwd=$(pwd)
 
 if [[ "$OS" == "Windows_NT" ]]; then
   docker_cmd="winpty docker"
+  if [[ "$MSYSTEM" == MINGW* ]]; then
+    pwd=`echo $pwd | sed 's/\//\/\//g'`
```

Comment:
Use `$()` instead of backticks.

Also, I think it's only the initial `/` that needs to be double?

E.g. `//foo/bar/baz`, so perhaps modify row 6 to have an initial `/` too? I can check on Mac if that works OK if needed.

Reply:
Changed to leading / in paths instead.